modules/fontrendering/ext/freetype/CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
-add_subdirectory(freetype)
+# Use our own packaging/install locations
+set(SKIP_INSTALL_ALL TRUE)
 
-#--------------------------------------------------------------------
-# Supress warnings
-ivw_suppress_compiler_warnings(freetype)
+add_subdirectory(freetype)
+ivw_get_targets_in_dir_recursive(targets ${CMAKE_CURRENT_SOURCE_DIR})
+ivw_suppress_compiler_warnings(${targets})
+set_target_properties(${targets} PROPERTIES MACOSX_RPATH ON)
```

Comment:
When do we need to do this?

Reply:
Did not look into it before but apparently it is when INSTALL_NAME_DIR is set for the project
https://blog.kitware.com/upcoming-in-cmake-2-8-12-osx-rpath-support/